test/groups/circle_group.jl

```diff
@@ -1,6 +1,10 @@
 include("../utils.jl")
 include("group_utils.jl")
 
+# TODO: remove after bug in StaticArray is fixed
+@inline Base.copy(a::SizedArray) = __copy(a)
+@inline __copy(a::SizedArray{S,T}) where {S,T} = SizedArray{S,T}(copy(a.data))
```

Comment:
just a short question - why the two `_`?

Reply:
No real reason, I just felt like it's less likely to extend a function we already have somewhere else (but most likely don't).

Reply:
Ok, well we could also just use `_copy` - just for clarity (others might also wonder why we use `__` here; but I leave this up to you :)

Reply:
I'm going to delete it when https://github.com/JuliaArrays/StaticArrays.jl/pull/1037 is merged anyway so it's just a temporary fix.